src/uu/split/src/filenames.rs

```diff
@@ -0,0 +1,533 @@
+//  * This file is part of the uutils coreutils package.
+//  *
+//  * For the full copyright and license information, please view the LICENSE
+//  * file that was distributed with this source code.
+// spell-checker:ignore zaaa zaab zzaaaa zzzaaaaa
+//! Compute filenames from a given index.
+//!
+//! The [`FilenameFactory`] can be used to convert a chunk index given
+//! as a [`usize`] to a filename for that chunk.
+//!
+//! # Examples
+//!
+//! Create filenames of the form `chunk_??.txt`:
+//!
+//! ```rust,ignore
+//! use crate::filenames::FilenameFactory;
```

Comment:
We can get rid of the `ignore`, here and in the other doctests. If we mark `filenames` as `pub mod filenames;` in `split.rs` and change this import to `use uu_split::filenames::FilenameFactory;`. That gives us more tests for free.

Reply:
That's good to know. I don't think it's worth exposing the module as public just to get these doctests to work, so I'll leave it for now. Thanks for informing me.